### PR TITLE
Filter module permission

### PR DESCRIPTION
**Pull request by vllorca**

### CODE REVIEW

File: terra_accounts/signals.py [hunk 1]
```diff
@@ -1,24 +1,22 @@
...
         ctype = ContentType.objects.get_for_model(TerraPermission)
 
         TerraPermission.objects.update_or_create(
-            content_type=ctype,
-            codename=perm,
-            defaults={
-                'name': name,
-                'module': module
-            }
+            content_type=ctype, codename=perm, defaults={"name": name, "module": module}
```
Comment (CFarcy) [inline terra_accounts/signals.py:21]: This change might not be needed

File: test_terra_accounts/settings.py [hunk 2]
```diff
@@ -146,3 +146,7 @@
     'JWT_PAYLOAD_HANDLER': 'terra_accounts.jwt_payload.terra_payload_handler',
     'JWT_EXPIRATION_DELTA': timedelta(seconds=9999)
 }
+
+TERRA_APPLIANCE_SETTINGS = {
+    'disabled_modules': [],
+}
```
Comment (CFarcy) [inline test_terra_accounts/settings.py:152]: It is better to have a new line at the end of a file (but not a big deal).

**Review by CFarcy — APPROVED** [review]

**Review by jrmi — APPROVED** [review]

File: terra_accounts/signals.py (hunk 1 above)
Comment (vllorca) [reply terra_accounts/signals.py]: I revert this change

File: test_terra_accounts/settings.py (hunk 2 above)
Comment (vllorca) [reply test_terra_accounts/settings.py]: I add a newline at the end of the file

**Pull request merged**